e2e-tests/py3-pandas-test.yaml

```diff
@@ -14,5 +14,15 @@ test:
       packages:
         - busybox
   pipeline:
+    - uses: python/import
```

Comment:
Great you picked the py3-pandas 🐼

I had started to work on this by adding a test, but instead of only doing the imports, decided to use one from their examples to get little more coverage. Backstory, I had created a PR that I used as an example for the melange docs, but spaced and never actually created the PR: https://github.com/chainguard-dev/melange/blob/main/docs/TESTING.md#test-environment-workspace

I am happy with the more comprehensive use of imports, but overall I'd like us to start testing little more functionality where possible.

Reply:
I think you *did* add such a test. See c219511b e2e-tests/test-fixtures/py3-pandas-test.py

I'm happy with multiple/redundant tests in melange that excercise it's functionality, even if they're overlapping in coverage.  We want to test the `imports` and this was an easy way to add that coverage.  @vaikas do you disagree with that?

Reply:
ha, indeed I did, thanks!! 🤣 Absolutely agree with adding more tests is better!!